### PR TITLE
Feature/user model

### PR DESCRIPTION
**Pull request by pawelkatny**
Added:
- user model with additional custom methods
- tests for user model

@IvanLiaskovski

### CODE REVIEW

File: src/helpers/dbServer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const { MongoMemoryServer } = require("mongodb-memory-server");
+const mongoose = require("mongoose");
+
+let db = undefined;
```
Comment (IvanLiaskovski) [inline src/helpers/dbServer.js:4]: You can skip adding 'undefined' by default to the db variable.

**Review by IvanLiaskovski — COMMENTED** [review]
I have a minor suggestion regarding the assignment of 'undefined' to the 'db' variable. 
Address this before merging the branches.

File: src/helpers/dbServer.js (hunk 1 above)
Comment (pawelkatny) [reply src/helpers/dbServer.js:4]: Typo or autocomplete involved. Fixed in the recent PR.

Comment (pawelkatny) [reply src/helpers/dbServer.js:4]: Closing this one.

**Pull request closed without merging**